saga/src/test/java/com/iluwatar/saga/choreography/SagaApplicationTest.java

```diff
@@ -32,9 +32,9 @@
 /***
  * empty test
  */
-public class SagaApplicationTest {
+class SagaApplicationTest {
   @Test
-  public void shouldExecuteWithoutException() {
+  void shouldExecuteWithoutException() {
     assertDoesNotThrow(() -> SagaApplication.main(new String[]{}));
   }
 }
```

Comment:
Please add an extra line here

Reply:
You want an empty line after the method declaration and before the assertion?

Reply:
Yes an empty line at the end of the file.

Reply:
Ah, duh. I need coffee. Updated!